silica/memory.py

```diff
@@ -5,3 +5,9 @@ def __init__(self, height, width):
 
     def __eq__(self, other):
         return isinstance(other, MemoryType) and self.height == other.height and self.width == other.width
+
+    def __str__(self):
+        return "<MemoryType: {}, {}>".format(self.height, self.width)
+
+    def __repr__(self):
+        return "<MemoryType: {}, {}>".format(self.height, self.width)
```

Comment:
We could just have `__str__` call `__repr__` since right now they're the same. In fact, I think if you only define `__repr__`, `__str__` will default to calling `__repr__`